frontend/src/views/WheresMyGeneV2/components/HeatMap/index.tsx

```diff
@@ -110,19 +110,20 @@ export default memo(function HeatMap({
   isScaled,
   cellTypeSortBy,
   geneSortBy,
-  selectedOrganismId,
   echartsRendererMode,
   allChartProps,
   setAllChartProps,
+  tissuesByName,
   setTissuesByName,
   expandedTissues,
   setExpandedTissues,
 }: Props): JSX.Element {
   const {
     xAxisHeight,
-    selectedFilters: { tissues: filteredTissues },
+    selectedFilters: { tissues: filteredTissueIds },
```

Comment:
make variable name explicit by suffixing with `Ids`. We actually have a share link breaking change that previously the share link tissue expects tissue name, but now it seems to expect tissue ids. Not sure if this is an intended change or a bug. Signe is out OOO, so we'll need to check when she comes back!